package.json

```diff
@@ -6,7 +6,8 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
+    "dev":
+      "cross-env NODE_ENV=development webpack-dev-server --host 0.0.0.0 --open --hot",
```

Comment:
`--open` flag can be removed since it'll try to open a browser inside the image.

Reply:
Yeah, I noticed the warning message but I was hoping to find a good way to fix this later.  The plan was to leave the error message so I'd remember to track that down.  Not sure if there's actually a way to get around it or not, though.  Might be better to just remove now.

Reply:
There's probably some docker magic that you can pass through a display driver (I think this was a problem we ran into back with dockerizing the camera stuff), but probably not worth the effort for local dev (imo) lol.

Reply:
Yeah, I agree that it's probably more work than it's worth haha.  Let's ditch it for now.